apis/hiveinternal/v1alpha1/clustersync_types.go

```diff
@@ -12,6 +12,8 @@ import (
 // +k8s:openapi-gen=true
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:path=clustersyncs,shortName=csync,scope=Namespaced
+// +kubebuilder:printcolumn:name="Status",type=string,JSONPath=`.status.conditions[0].reason`
+// +kubebuilder:printcolumn:name="Message",type=string,JSONPath=`.status.conditions[0].message`
```

Comment:
I assume these are just blank if there's no `.status.conditions[0]`. But afaict there always is.